asserts/assertstest/assertstest.go

```diff
@@ -266,7 +273,19 @@ type StoreStack struct {
 }
 
 // NewStoreStack creates a new store assertion stack. It panics on error.
-func NewStoreStack(authorityID string, rootPrivKey, storePrivKey asserts.PrivateKey) *StoreStack {
+// optional privKeys can be in order: root, store, generic
+func NewStoreStack(authorityID string, privKeys ...asserts.PrivateKey) *StoreStack {
+	if len(privKeys) > 3 {
+		panic("too many private keys specified, expected at most: root, store, generic")
+	}
+	for len(privKeys) < 3 {
+		privKey, _ := GenerateKey(752)
+		privKeys = append(privKeys, privKey)
+	}
+	rootPrivKey := privKeys[0]
+	storePrivKey := privKeys[1]
+	genericPrivKey := privKeys[2]
```

Comment:
That seems somewhat deceiving indeed. The signature does make it look like the order is irrelevant. The alternative of just adding another genericPrivKey parameter and mechanically make it nil by default would be better, but still error prone.

Perhaps we should introduce a StoreKeys type with explicitly named fields, and define only the ones we care about.